src/chains/ethereum/ethereum/tests/provider.test.ts

```diff
@@ -141,6 +141,83 @@ describe("provider", () => {
       await provider.disconnect();
     });
 
+    describe("uses timestamp adjustment in subsequent blocks after calling `evm_mine` with a `timestamp` argument", () => {
+      const timestampIncrement = 5; // seconds
+      const timeArgumentSeconds = 100;
+
+      async function mineBlocksForTimestamps(
+        provider: EthereumProvider
+      ): Promise<string[]> {
+        // mine a block with a specified timestamp
+        await provider.request({
+          method: "evm_mine",
+          params: [timeArgumentSeconds]
+        });
+        const specifiedBlock = await provider.request({
+          method: "eth_getBlockByNumber",
+          params: ["latest", false]
+        });
+        // mine a block without a specified timestamp
+        await provider.request({
+          method: "evm_mine",
+          params: []
+        });
+        const unspecifiedBlock = await provider.request({
+          method: "eth_getBlockByNumber",
+          params: ["latest", false]
+        });
+
+        return [specifiedBlock.timestamp, unspecifiedBlock.timestamp];
+      }
+      it("should work with `timestampIncrement` of `clock`", async () => {
+        const provider = await getProvider({
+          miner: { timestampIncrement: "clock" }
+        });
+
+        const [specifiedBlock, unspecifiedBlock] =
+          await mineBlocksForTimestamps(provider);
+
+        assert.strictEqual(
+          specifiedBlock,
+          `0x${timeArgumentSeconds.toString(16)}`,
+          "Unexpected timestamp for block mined with specified timestamp"
+        );
+
+        assert(
+          +unspecifiedBlock >= timeArgumentSeconds &&
+            +unspecifiedBlock <= timeArgumentSeconds + 1000,
+          `Unexpected timestamp for block mined without specified timestamp - expected a value between ${timeArgumentSeconds} and ${
+            timeArgumentSeconds + 1000
+          }, got ${+unspecifiedBlock}`
+        );
+
+        await provider.disconnect();
+      });
+
+      it("should work with a numeric `timestampIncrement`", async () => {
+        const provider = await getProvider({
+          miner: { timestampIncrement }
```

Comment:
Since this `timestampIncrement` is only used in this function is there a reason to define it in the `describe`?